components/cam/src/physics/cam/clubb_intr.F90

```diff
@@ -2138,15 +2140,19 @@ real(r8) :: ugust  ! function: gustiness as a function of convective rainfall
       wv_a = 0._r8
       wl_a = 0._r8
       do k=1,pver
-         clubb_s(k) = cpair*((thlm(i,k)+(latvap/cpair)*rcm(i,k))/exner_clubb(i,k))+ &
-                      gravit*state1%zm(i,k)+state1%phis(i)
-         se_a(i) = se_a(i) + clubb_s(k)*state1%pdel(i,k)*invrs_gravit
+         enthalpy = cpair*((thlm(i,k)+(latvap/cpair)*rcm(i,k))/exner_clubb(i,k))
+         clubb_s(k) = enthalpy + gravit*state1%zm(i,k)+state1%phis(i)
+!         se_a(i) = se_a(i) + clubb_s(k)*state1%pdel(i,k)*invrs_gravit
+         se_a(i) = se_a(i) + enthalpy * state1%pdel(i,k)*invrs_gravit
```

Comment:
is it correct to include the latent heat (vapor) in the enthalpy?
it also may be confusing to use "se_a", "se_b", which suggests static energy, but here in this code it is no longer static energy?

Reply:
In the same file
thlm(i,k)    = state1%t(i,k)*exner_clubb(i,k)-(latvap/cpair)*state1%q(i,k,ixcldliq)
and
rcm is cloud water mixing ratio (which i think what q(,,ixcldliq) is)
and so
enthalpy=cpair*((thlm(i,k)+(latvap/cpair)*rcm(i,k))/exner_clubb(i,k))
=cpair*state1%t ?
For se_a and se_b, i was trying to keep number of changes minimal.